STAR/draft-ietf-acme-star.md

```diff
@@ -459,6 +469,34 @@ The server MUST NOT issue any additional certificates for this order beyond its
 
 Immediately after the order expires, the server MUST respond with 403 (Forbidden) to any requests to the certificate endpoint.  The response SHOULD provide additional information using a problem document {{RFC7807}} with type "urn:ietf:params:acme:error:recurrentOrderExpired".
 
+## Negotiate unauthenticated GET
+{: #certificate-get-nego }
+
+In order to enable the name delegation workflow defined in
+{{I-D.sheffer-acme-star-request}} as well as to increase the reliability of the
+STAR ecosystem (see {{dependability}} for details), this document defines a
+mechanism that allows a server to advertise support for accessing certificate
+resources via unauthenticated GET (instead of, or in addition to, POST-as-GET),
+and a client to enable this service with per-Order granularity.
+
+Specifically, a server states its availability to grant unauthenticated access
+to a client's Order certificate(s) by setting the star-allow-certificate-get
+key to true in the meta field of the Directory object:
+
+- star-allow-certificate-get (optional, boolean): If this field is present and
+  set to true, the server allows GET requests to certificate URLs.
+
+A client states her will to access the issued certificate(s) via
+unauthenticated GET by adding a recurrent-certificate-get key to her Order and
+setting it to true.
+
+- recurrent-certificate-get (optional, boolean): If this field is present and
+  set to true, the client requests the server to allow unauthenticated GET to
+  the certificate(s) associated with this Order.
+
+If the server accepts the request, it MUST reflect the key in the Order.  If it
+doesn't, it SHOULD return an error response.
```

Comment:
I wonder about the error response (though I was in favor before): if this flag is used for reliability, not just for delegation, then fallback to POST-as-GET is reasonable. For delegation of course, this fallback would not be useful.

Reply:
I agree.  I'm going to drop the "SHOULD return error".